src/main/java/bogdanSandu/driver/BrowserManager.java

```diff
@@ -1,4 +1,4 @@
-package bogdanSandu.driver;
+package BogdanSandu.driver;
```

Comment:
Packages in Java should start with lower and not capital letters so please leave the package name to bogdanSandu. Also do this change for the package inside test/java. Thanks!

Reply:
Please use lowerCase when naming packages as per Java conventions. Thanks!